Migrating OpenGeo Explorer docs to User Manual

bmmpxf opened the pull request.

jodygarnett commented on docs/usermanual/source/qgis/explorer/tutorial/index.rst:
```diff
@@ -0,0 +1,285 @@
+
+   It contains the following layers:
+
+   * 
```
Based on the picture I would expect this list to be:

```
* elevation
* geology_alnduse
* dem
```

jodygarnett commented on docs/usermanual/source/qgis/explorer/install.rst:
```diff
@@ -0,0 +1,43 @@
+
+      New plugin repository added successfully
+
+#. With the Boundless repository added, the OpenGeo Explorer plugin will not be available to be installed. Click :guilabel:`Not Installed` to see a list of all plugins.
```
This sentence is difficult to understand and confused me (in part because as a mac user the plugin was already installed - just not enabled).

```
With the Boundless repository added details of the :guilabel:`OpenGeo Explorer` are known. Since we have not yet installed the plugin it is listed on the :guilabel:`Not Installed` page. Click :guilabel:`Not Installed` to see a list of all plugins available for installation.
```

volaya replied:
this is a typo: It should say "will *now* be available to be installed"

jodygarnett commented on docs/usermanual/source/qgis/explorer/install.rst:
```diff
@@ -0,0 +1,43 @@
+
+This section will describe how to install the OpenGeo Explorer plugin.
+
+The OpenGeo Explorer plugin requires **QGIS version 2.8 or higher** to run, and is installed using the QGIS Plugin Manager.
```
This step is only required for linux users as mac and windows users have the plugin included (but not enabled) by default.

This section should be broken into two:
a) installing - for linux users
b) enabling - for all users

jodygarnett commented on docs/usermanual/source/qgis/explorer/install.rst:
```diff
@@ -0,0 +1,43 @@
+
+      OpenGeo Explorer in the list of plugins
+
+#. The plugin will be installed. To verify that installation was successful, you should see a new menu in the QGIS menu bar called :guilabel:`OpenGeo`.
```
This is not the case, need to find the plugin in the *Installed* list and enable it before the menu will show up.

jodygarnett commented on docs/usermanual/source/qgis/explorer/install.rst:
```diff
@@ -0,0 +1,43 @@
+
+   .. figure:: actions/img/plugin_install.png
+
+      OpenGeo Explorer in the list of plugins
```
This image is of "All" plugins, while the instructions took me to "Not Installed".

bmmpxf commented:
Thanks @jodygarnett . I think I addressed all of your feedback aside from the vagaries of installs on different operating systems. But let's save that for a separate ticket.

@volaya , any objections to merging this in?

volaya commented:
No objections, this looks good to me.
